CONTRIBUTING.md

```diff
@@ -49,6 +55,33 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://github.com/kubernetes/community/tree/master/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Reviewing a Patch
+
+### Approvals
+
+Please see the [Kubernetes community document on pull
+requests](https://git.k8s.io/community/contributors/guide/pull-requests.md) for more information about the merge
+process.
+
+- A PR is approved by one of the project maintainers and owners after reviews.
+- Approvals should be the very last action a maintainer takes on a pull request.
+
+## Reviews
+
+> Parts of the following content have been adapted from https://google.github.io/eng-practices/review.
+
+Any Kubernetes organization member can leave reviews and `/lgtm` a pull request.
+
+Code reviews should generally look at:
```

Comment:
xposting https://github.com/kubernetes-sigs/cluster-api/pull/3194#discussion_r440421253

I would refer to "REVIEWING.md" for details about what a review should look at

Reply:
These bits go in the book as well, let's have the generic high level list here; I'll add a link to the REVIEWING doc once it merges